terraform/10-video-hearing-wowza/modules/wowza/wowza_lb_alerts.tf

```diff
@@ -0,0 +1,51 @@
+data "azurerm_monitor_action_group" "dev" {
+  resource_group_name = "vh-infra-core-${env}"
```

Comment:
missed the var. here

Reply:
and env is not a variable, it is var.environment on this module

Reply:
Changes made